deno/payloads/v10/user.ts

```diff
@@ -145,6 +145,10 @@ export enum UserFlags {
 	 * User has been identified as spammer
 	 */
 	Spammer = 1 << 20,
+	/**
+	 * User's account has been quarantined based on recent activity
+	 */
+	Quarantined = 1 << 44
```

Comment:
This is invalid, `1 << 44` yields the same value as `1 << 12`.

Reply:
Is it bigints time? 🥲

Reply:
The client uses `Math.pow(2, 44)`

Reply:
That's webpack mostly. We should still aim to use bitshifting

Reply:
It should also use `@unstable`, since it wasn't documented by Discord

Reply:
> It should also use `@unstable`, since it wasn't documented by Discord

I think it wouldn't be documented in Discord API Docs, in the same way as the `Spammer` user flag.

I just have this reference: https://support.discord.com/hc/en-us/articles/6461420677527

Reply:
The Spammer userflag should have an `@unstable` comment as well

Reply:
> The client uses `Math.pow(2, 44)`

Well, that does work:
![image](https://user-images.githubusercontent.com/24852502/174366781-35f34311-a49d-45cf-959f-e6d43d6f4c4e.png)

However, it's a bit useless as integers, because you can't bitshift bits higher than 32 bits.

Reply:
Let's use the Math.pow for now (or directly the number for the bit) and when Discord moves to bigints, so will we

Reply:
Done.